src/components-examples/material/datepicker/datepicker-dialog/datepicker-dialog-example-dialog.html

```diff
@@ -0,0 +1,13 @@
+<h2 mat-dialog-title>Datepicker in a Dialog</h2>
+<mat-dialog-content align="center">
+  <mat-form-field>
+    <mat-label>Select a date</mat-label>
+    <input matInput [matDatepicker]="picker" [formControl]="date">
+    <mat-datepicker-toggle matIconSuffix [for]="picker"></mat-datepicker-toggle>
+    <mat-datepicker #picker></mat-datepicker>
+  </mat-form-field>
+</mat-dialog-content>
+<mat-dialog-actions>
+  <button mat-button (click)="onClear()">Clear</button>
```

Comment:
I see that this example is based on the "Dialog Overview", however using `mat-dialog-close` should reduce the unrelated code even more.
```suggestion
  <button mat-button mat-dialog-close>Clear</button>
```
This would eliminate the need for the `onClear` method.

Reply:
Thanks, I updated it.